webbpsf/tests/test_trending.py

```diff
@@ -0,0 +1,35 @@
+import astropy
+import astropy.time
+import astropy.units as u
+import calendar
+import webbpsf
+
+def get_month_start_end(year, month):
+    _, ndays = calendar.monthrange(year, month)
+    start_date_str = f"{year:04d}-{month:02d}-01"
+    end_date_str = f"{year:04d}-{month:02d}-{ndays:02d}"
+
+    start_date = astropy.time.Time(start_date_str)
+    end_date = astropy.time.Time(end_date_str)
+
+    return start_date, end_date
```

Comment:
This looks like an exact copy of the `get_month_start_end` function from webbpsf.trending? Why is it useful to duplicate that function here into the test code, instead of just calling the existing function?

Reply:
Good catch, it wasn't deliberate, I forgot to swap that out.  Thank you!